src/lambda_codebase/account_processing/configure_account_alias.py

```diff
@@ -22,8 +22,13 @@ def create_account_alias(account, iam_client):
     )
     try:
         iam_client.create_account_alias(AccountAlias=account.get("alias"))
-    except iam_client.exceptions.EntityAlreadyExistsException:
-        pass
+    except iam_client.exceptions.EntityAlreadyExistsException as error:
+        LOGGER.error(
+            """The account alias security already exists.
+            The account alias must be unique across all Amazon Web Services products.
+            Refer to https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html#AboutAccountAlias"""
+        )
```

Comment:
Could you ensure this shows the alias that is in use?
At the moment it refers to the `security` alias :)

Plus could you convert this to the multi-line strings that we use elsewhere, like this?

```suggestion
        LOGGER.error(
            f"The account alias ${account.get('alias')} already exists."
            "The account alias must be unique across all Amazon Web Services products."
            "Refer to https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html#AboutAccountAlias"
        )
```

Reply:
Thanks for spotting @sbkok , I've committed a fix.